[DOC-10075] Document new topology minimums for Advanced (#18880)
cockroachdb/docs: src/current/cockroachcloud/ccloud-get-started.md

Reviewer: General comment on all of these - do we have confirmation that plan names are not changing in ccloud. I assumed the command may not change for continuity, but surprised to see the response also referencing the old plan names

Author: `ccloud` has not been updated yet so the related docs have not been updated yet. This is getting merged into an integration branch for now.

ccloud cluster create dedicated
~~~

> This command creates a 3-node CockroachDB {{ site.data.products.advanced }} cluster with 4 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.

Reviewer: Note: This command has not been updated yet with the default vCPUs set to 4. But we do plan to make this change. I'll circle back to confirm.

ccloud cluster create dedicated
~~~

> This command creates a 3-node CockroachDB {{ site.data.products.advanced }} cluster with 4 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.

Author: ```suggestion
This command creates a 1-node CockroachDB {{ site.data.products.advanced }} cluster with 4 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.
```

The `ccloud` commands have not yet been updated. I'm not sure if the tier names are being updated or we are going with small-s serverless and small-d dedicated.

You can set the cluster name, cloud infrastructure provider, region, number of nodes, and storage as command options. The following command is equivalent to the previous command that uses the default values.

{% include_cached copy-clipboard.html %}
~~~ shell

> ccloud cluster create dedicated blue-dog us-central1:3 --cloud GCP --vcpus 4 --storage-gib 110

Author: ```suggestion
ccloud cluster create dedicated blue-dog us-central1:1 --cloud GCP --vcpus 4 --storage-gib 110
```

The `id` in the output is the cluster ID. You use the `name` in other `ccloud` commands to identify the cluster on which the `ccloud` command operates.

When creating multi-region clusters, you must specify how many nodes should be in each region supported by the cloud infrastructure provider. For example, the following command creates a 12-node cluster where 8 nodes are in `us-central1` and 4 nodes are in `us-west2`. For optimum performance, it is generally recommended to configure the same number of nodes in each region.

{% include_cached copy-clipboard.html %}
~~~ shell
ccloud cluster create dedicated blue-dog us-central1:8 us-west2:4 --cloud GCP --vcpus 4 --storage-gib 110
~~~
</section>

{{site.data.alerts.callout_info}}

> To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.basic }} cluster, or create a CockroachDB {{ site.data.products.advanced }} cluster, you must [add a credit card](billing-management.html) to your organization.

Reviewer: This should be updated. 
- You can't set RUs anymore for Standard clusters. 
- For new orgs, you can't create any cluster without a credit card.

Author: I think for the free trial period, you do not yet need to add a credit card. Can you confirm?

The `id` in the output is the cluster ID. You use the `name` in other `ccloud` commands to identify the cluster on which the `ccloud` command operates.

When creating multi-region clusters, you must specify how many nodes should be in each region supported by the cloud infrastructure provider. For example, the following command creates a 12-node cluster where 8 nodes are in `us-central1` and 4 nodes are in `us-west2`. For optimum performance, it is generally recommended to configure the same number of nodes in each region.

{% include_cached copy-clipboard.html %}
~~~ shell
ccloud cluster create dedicated blue-dog us-central1:8 us-west2:4 --cloud GCP --vcpus 4 --storage-gib 110
~~~
</section>

{{site.data.alerts.callout_info}}

> To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.basic }} cluster, or create a CockroachDB {{ site.data.products.advanced }} cluster, you must [add a credit card](billing-management.html) to your organization.

Author: ```suggestion
To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.basic }} cluster, or to create a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.advanced }} cluster, you must [add a credit card](billing-management.html) to your organization.
```

> To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.basic }} cluster, or to create a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.advanced }} cluster, you must [add a credit card](billing-management.html) to your organization.

Author: ```suggestion
To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.basic }} cluster, scale a CockroachDB {{ site.data.products.standard }}'s nodes, or create a {{ site.data.products.advanced }} cluster, you must [add a payment method](billing-management.html) to your organization.
```

> To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.basic }} cluster, or to create a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.advanced }} cluster, you must [add a credit card](billing-management.html) to your organization.
> {{site.data.alerts.end}}

Author: ```suggestion
```
Removing this admonition from the `ccloud` command page since you can do all this stuff with free trial credits.
